services/web/client/source/class/osparc/component/widget/NodesTree.js

```diff
@@ -314,11 +329,31 @@ qx.Class.define("osparc.component.widget.NodesTree", {
           this.__deleteNode();
         }
       }, this);
+
       this.addListener("keypress", function(keyEvent) {
         if (keyEvent.getKeyIdentifier() === "F2") {
           this.__openItemRenamer();
         }
       }, this);
+
+      this.__toolBar.addListener("resize", () => {
+        const toolBarBtns = this.__getToolbarButtons(this.__toolBar);
+        if (this.__toolBar.getBounds().width < this.__toolbarInitMinWidth) {
+          // Hide Label
+          toolBarBtns.forEach(toolBarBtn => {
+            const label = toolBarBtn.getChildControl("label");
+            label.exclude();
+            toolBarBtn.setToolTipText(label.getValue());
+          });
+        } else {
+          // Show Label
+          toolBarBtns.forEach(toolBarBtn => {
+            toolBarBtn.getChildControl("label").show();
+            toolBarBtn.setToolTipText(null);
```

Comment:
After thinking about this it makes sense, no same tooltip when text is visible, is not useful.